ide/.merlin.in

```diff
@@ -1,8 +1,10 @@
 PKG unix laglgtk3 lablgtk3-sourceview3 
 
-S utils
-B utils
-S protocol
-B protocol
+S coqide/utils
+B coqide/utils
+S coqide/protocol
+B coqide/protocol
+S coqide/
+B coqide/
```

Comment:
This seems relevant for a specific PR.

Reply:
It is a separate commit, making it easy to revert if there's any problem.  Does it really need to be a separate PR?  Is the extra overhead worth it?

Reply:
> Is the extra overhead worth it?

I don't know. I believe that it is generally preferred to have PRs bound to semantic units (for further reference) but I agree that this is extra burden. On my side, I'm ok to keep it here, but others might insist for a different PR.